frontends/web/src/locales/en/app.json

```diff
@@ -1172,7 +1172,6 @@
       "placeholder": "Enter hexadecimal data"
     },
     "error": {
-      "feeTooLow": "fee too low",
```

Comment:
why is feeTooLow gone again?

Reply:
hmm good question, i guess we pushed locize from master while this was in the staging branch :/ will fix

Reply:
First: https://github.com/digitalbitbox/bitbox-wallet-app/pull/987